test/api/v3/integration/tasks/challenges/POST-tasks_challenges_challengeId_tasks_id_score_direction.test.js

```diff
@@ -82,14 +82,14 @@ describe('POST /tasks/:id/score/:direction', () => {
     });
 
     it('should update the history', async () => {
+      await user.post(`/tasks/${usersChallengeTaskId}/score/down`);
       await user.post(`/tasks/${usersChallengeTaskId}/score/up`);
 
       let tasks = await user.get(`/tasks/challenge/${challenge._id}`);
       let task = find(tasks, {_id: daily._id});
 
       expect(task.history).to.have.lengthOf(1);
       expect(task.history[0].date).to.not.equal(previousTaskHistory.date);
-      expect(task.history[0].value).to.not.equal(previousTaskHistory.value);
```

Comment:
Possible, the only way to test this is by somehow changing the time to tomorrow so that the dailies are unchecked. Can cron be used?

Reply:
Possibly you could use the approach the debug menu does, by changing `lastCron` into the past and rerunning cron?

Reply:
I will try that

Reply:
I attempted to replace

```JavaScript
await user.post(`/tasks/${usersChallengeTaskId}/score/down`);
```

with

```JavaScript
      let newCron = new Date(2015, 11, 20);
      await user.post('/debug/set-cron', {
          lastCron: newCron,
      });

      await user.sync();
      expect(user.lastCron).to.eql(newCron);
```

but for some reason, this fails to update the `lastCron`. Then, I tried running the tests in `test/api/v3/integration/debug` but they fail too. Do I have to carry out any extra step before running those tests?

Reply:
Hmm, that's peculiar. Do the tests fail for you locally even without the above change? That suggests a dev environment issue, given that the tests are passing on CI.

Does it behave any differently if you set `lastCron` directly as part of the test rather than using the debug API?

Reply:
I made two mistakes. 1) Did not send a request to /cron 2) Forgot to quit the server before running the tests.

After fixing those two, the tests have passed successfully.